app/templates/suppliers/details.html

```diff
@@ -100,11 +100,11 @@
   {% if supplier.registrationCountry and not supplier.companiesHouseNumber %}
     {% call summary.row() %}
       {{ summary.field_name('Where your business was established') }}
-      {{ summary.text(supplier.registrationCountry|upper) }}
+      {{ summary.text(country_name) }}
     {% endcall %}
   {% endif %}
 
-  {% if supplier.otherCompanyRegistrationNumber and not supplier.companiesHouseNumber and supplier.registrationCountry != 'gb' %}
+  {% if supplier.otherCompanyRegistrationNumber and not supplier.companiesHouseNumber and country_name != 'United Kingdom' %}
```

Comment:
Minor thing but it feels like it would be nice to have the logic based on the value in supplier rather than on the derived country name if possible - something like `supplier.registrationCountry != 'country:gb'?

Ah, but then that won't cover the "legacy" case of just `gb` will it?

OK probably fine like this then.